benchmarks/postprocess-plot.py

```diff
@@ -156,9 +156,9 @@
       plot(y/slope2,y,'k-',label='%g iter/s'%(slope2/vdim))
 
    # Plot information
-   title('(%i node%s, %i ranks), %s, %s'%(
+   title(r'%i node%s $\times$ %i ranks, %s, %s'%(
```

Comment:
Right now we have total ranks rather than ranks per process, so times doesn't make sense here. Do we want total ranks or ranks per node? Nodes x Ranks Per Node = Total Ranks takes up a lot of space, which is why I had left it as (Nodes, Total Ranks).

Reply:
I slightly prefer `nodes \times ranks_per_node` because mental multiplication is easier than division, resources like memory bandwidth are largely shared on each node, and architectures can have a pretty big range in number of cores and relative power of those cores.

Reply:
Updated to ranks per node. I think we're good to merge this now.